### PR TITLE
Integration tests

### LINKED ISSUE #3: Create integration tests a la fal-ai/fal
check https://github.com/fal-ai/fal/blob/main/.github/workflows/integration_tests.yml

### PR DESCRIPTION
**Pull request by mederka**
closes #3

### CODE REVIEW

File: .github/workflows/integration_tests.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ["3.8.x"]
+        # TODO: change to 1.0.1 when available
+        dbt: ["1.0.1rc1"]
```
Comment (chamini2) [inline .github/workflows/integration_tests.yml]: We can do 1.0.4, maybe we should change in fal-ai/fal too?

File: .github/workflows/integration_tests.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+
+      - name: Fix jinja
+        run: |
+          pip install markupsafe==2.0.1
```
Comment (chamini2) [inline .github/workflows/integration_tests.yml]: maybe not necessary with 1.0.4

File: .github/workflows/integration_tests.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+          pip install fal
+
+      - name: Install dbt
+        run: |
+          if [[ '${{ matrix.dbt }}' =~ ^0.*$ ]]
+          then
+            pip install dbt==${{ matrix.dbt }}
+          else
+            pip install dbt-postgres==${{ matrix.dbt }}
+          fi
```
Comment (chamini2) [inline .github/workflows/integration_tests.yml]: No need of `if-then-else`, just handling `1.X` versions which should install `dbt-postgres`

File: .github/workflows/integration_tests.yml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        with:
+          python-version: ${{ matrix.python }}
+
+      - name: Fix jinja
+        run: |
+          pip install markupsafe==2.0.1
+
+      - name: Install fal
+        run: |
+          pip install fal
```
Comment (chamini2) [inline .github/workflows/integration_tests.yml:28]: I think we don't use this, right?

Comment (mederka) [reply .github/workflows/integration_tests.yml:28]: We're using fal to validate model contents by comparing dataframes.

**Review by chamini2 — APPROVED** [review]
Great solutin

**Pull request merged**